test/system/030-run.bats

```diff
@@ -58,6 +58,25 @@ echo $rand        |   0 | $rand
     is "$output" ".*invalidflag" "failed when passing undefined flags to the runtime"
 }
 
+# bats test_tags=ci:parallel
+@test "podman run - containers.conf runtime options" {
+    skip_if_remote "requires local containers.conf"
+
+    containersConf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersConf <<EOF
+[engine]
+runtime="crun"
+
+[engine.runtimes_flags]
+crun = [
+  "invalidflag",
+]
```

Comment:
nit testing wise it would be nice to define some other runtimes and ensure flags are indeed only set for each specific runtime.

Though given the "urgency" for this I am fine with the test as is minus the other comment